app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -175,6 +199,86 @@ private fun MessageHeader(messageHeader: MessageHeader) {
     }
 }
 
+@Composable
+private fun MessageFooter(
+    messageFooter: MessageFooter,
+    onReactionClicked: (String, String) -> Unit
+) {
+    FlowRow(
+        mainAxisSpacing = 4.dp,
+        crossAxisSpacing = 6.dp,
+        modifier = Modifier.padding(vertical = 4.dp)
+    ) {
+        messageFooter.reactions.entries
+            .sortedBy { it.key }
+            .forEach {
+                val reaction = it.key
+                val count = it.value
+                Pill(
+                    emoji = reaction,
+                    count = count,
+                    isOwn = messageFooter.ownReactions.contains(reaction),
+                    onTap = {
+                        onReactionClicked(messageFooter.messageId, reaction)
+                    }
+                )
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun Pill(
+    emoji: String,
+    count: Int,
+    isOwn: Boolean,
+    onTap: () -> Unit
+) {
+
+    val strokeColor = if (isOwn) {
+        MaterialTheme.wireColorScheme.secondaryButtonSelectedOutline
+    } else {
+        MaterialTheme.wireColorScheme.primaryButtonDisabled
+    }
+
+    val backgroundColor = if (isOwn) {
+        MaterialTheme.wireColorScheme.secondaryButtonSelected
+    } else {
+        MaterialTheme.wireColorScheme.surface
+    }
+
+    val textColor = if (isOwn) {
+        MaterialTheme.wireColorScheme.primary
+    } else {
+        MaterialTheme.wireColorScheme.labelText
+    }
+
+    CompositionLocalProvider(
+        LocalMinimumTouchTargetEnforcement provides false
+    ) {
+        OutlinedButton(
+            onClick = onTap,
+            border = BorderStroke(1.dp, strokeColor),
+            shape = RoundedCornerShape(12.dp),
+            colors = ButtonDefaults.outlinedButtonColors(containerColor = backgroundColor),
+            contentPadding = PaddingValues(start = 8.dp, end = 8.dp, top = 4.dp, bottom = 4.dp),
+            modifier = Modifier
+                .defaultMinSize(minWidth = 1.dp, minHeight = 1.dp)
+        ) {
+            Text(
+                emoji,
+                style = TextStyle(fontSize = 12.sp)
+            )
+            Text(
+                count.toString(),
+                modifier = Modifier.padding(start = 4.dp, end = 0.dp, top = 0.dp, bottom = 0.dp),
```

Comment:
same here can we not use magic numbers

Reply:
Yep! Forgot during the clean-up :P 

I'll address this now.